cdc/entry/mounter_test.go

```diff
@@ -423,3 +425,308 @@ func walkTableSpanInStore(t *testing.T, store tidbkv.Storage, tableID int64, f f
+				FieldType: types.FieldType{
+					Tp:   mysql.TypeYear,
+					Flag: mysql.NotNullFlag,
+				},
+			},
+			Res: int64(0),
+		},
+		// mysql.TypeNewDate
+		{
+			ColInfo: timodel.ColumnInfo{
+				FieldType: types.FieldType{
+					Tp:   mysql.TypeNewDate,
+					Flag: mysql.NotNullFlag,
+				},
+			},
+			Res: nil, // [TODO] seems not support by TiDB, need check
+		},
+		// mysql.TypeVarchar
+		{
+			ColInfo: timodel.ColumnInfo{
+				FieldType: types.FieldType{
+					Tp:   mysql.TypeVarchar,
+					Flag: mysql.NotNullFlag,
+				},
+			},
+			Res: []byte{},
+		},
+		// mysql.TypeTinyBlob
+		{
+			ColInfo: timodel.ColumnInfo{
+				FieldType: types.FieldType{
+					Tp:   mysql.TypeTinyBlob,
+					Flag: mysql.NotNullFlag,
+				},
+			},
+			Res: []byte{},
+		},
+		// mysql.TypeMediumBlob
+		{
+			ColInfo: timodel.ColumnInfo{
+				FieldType: types.FieldType{
+					Tp:   mysql.TypeMediumBlob,
+					Flag: mysql.NotNullFlag,
+				},
+			},
+			Res: []byte{},
+		},
+		// mysql.TypeLongBlob
+		{
+			ColInfo: timodel.ColumnInfo{
+				FieldType: types.FieldType{
+					Tp:   mysql.TypeLongBlob,
+					Flag: mysql.NotNullFlag,
+				},
+			},
+			Res: []byte{},
+		},
+		// mysql.TypeBlob
+		{
+			ColInfo: timodel.ColumnInfo{
+				FieldType: types.FieldType{
+					Tp:   mysql.TypeBlob,
+					Flag: mysql.NotNullFlag,
+				},
+			},
+			Res: []byte{},
+		},
+		// mysql.TypeVarString
+		{
+			ColInfo: timodel.ColumnInfo{
+				FieldType: types.FieldType{
+					Tp:   mysql.TypeVarString,
+					Flag: mysql.NotNullFlag,
+				},
+			},
+			Res: []byte{},
+		},
+		// mysql.TypeString
+		{
+			ColInfo: timodel.ColumnInfo{
+				FieldType: types.FieldType{
+					Tp:   mysql.TypeString,
+					Flag: mysql.NotNullFlag,
+				},
+			},
+			Res: []byte{},
+		},
+		// mysql.TypeBit
+		{
+			ColInfo: timodel.ColumnInfo{
+				FieldType: types.FieldType{
+					Flag: mysql.NotNullFlag,
+					Tp:   mysql.TypeBit,
+				},
+			},
+			Res: uint64(0),
+		},
+		// mysql.TypeJSON
+		{
+			ColInfo: timodel.ColumnInfo{
+				FieldType: types.FieldType{
+					Tp:   mysql.TypeJSON,
+					Flag: mysql.NotNullFlag,
+				},
+			},
+			Res: "null",
+		},
+		/*
+			// mysql.TypeEnum, panic, need fix
+			{
+				ColInfo: timodel.ColumnInfo{
+					FieldType: types.FieldType{
+						Tp:   mysql.TypeEnum,
+						Flag: mysql.NotNullFlag,
+					},
+				},
+				Res: int64(0),
+			},
+		*/
+		// mysql.TypeSet
+		{
+			ColInfo: timodel.ColumnInfo{
+				FieldType: types.FieldType{
+					Tp:   mysql.TypeSet,
+					Flag: mysql.NotNullFlag,
+				},
+			},
+			Res: uint64(0),
+		},
+		// mysql.TypeGeometry
+		{
+			ColInfo: timodel.ColumnInfo{
+				FieldType: types.FieldType{
+					Tp:   mysql.TypeGeometry,
+					Flag: mysql.NotNullFlag,
+				},
+			},
+			Res: nil,
+		},
+	}
+	testGetDefaultZeroValue(t, colAndRess)
+}
+
+func testGetDefaultZeroValue(t *testing.T, colAndRess []columnInfoAndResult) {
+	for _, colAndRes := range colAndRess {
+		val, _, _, _ := getDefaultOrZeroValue(&colAndRes.ColInfo)
```

Comment:
Better to check the other return values

Reply:
May need more unit tests also. I will add more in other pr.